templates/template.html

```diff
@@ -49,6 +49,9 @@
         <li class="nav-item">
           <a class="nav-link" href="/donation">Donation</a>
                </li>
+           <li class="nav-item">
+          <a class="nav-link" href="/recent">recently_uploaded</a>
```

Comment:
@sanoussibarkay 

Please replace `recently_uploaded` with `Recent articles`.

Why?

We do not want to show end users a variable name.


Hui